src/test/java/com/wikia/webdriver/testcases/templateclassification/TemplateClassificationTest.java

```diff
@@ -0,0 +1,52 @@
+package com.wikia.webdriver.testcases.templateclassification;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.helpers.Browser;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.oasis.components.templateclassificiation.TemplateClassification;
+
+import org.testng.annotations.Test;
+
+@Execute(asUser = User.STAFF, onWikia = "aga")
+@InBrowser(browser = Browser.CHROME)
+public class TemplateClassificationTest extends NewTestTemplate {
+
+  private Navigate navigate;
+  private TemplateClassification templateClassification;
+
+  private void init() {
+    this.navigate = new Navigate(driver);
+    this.templateClassification = new TemplateClassification(driver);
+  }
+
+  @Test(groups = "templateClassification_openAndClose")
+  public void templateClassification_openAndClose() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)